openlayers/src/index.js

```diff
@@ -78,14 +78,11 @@ export class PMTilesVectorSource extends VectorTile {
   };
 
   constructor(options) {
-    super({
-      ...options,
-      ...{
-        state: "loading",
-        url: "pmtiles://" + options.url + "/{z}/{x}/{y}",
-        format: new MVT(),
-      },
-    });
+    options.state = "loading"
+    options.url = "pmtiles://" + options.url + "/{z}/{x}/{y}"
```

Comment:
This doesn't seem to work because line 88, the first argument to `pmtiles.FetchSource` needs to be the plain URL `https://example.com/name.pmtiles` instead of the prefixed one that `PMTilesVectorSource` expects

Should we leave the code like before but just change l86 to: `format: options.format || new MVT(),` ?

Thanks

Reply:
Good point. Changed to old version. Only `format` has a fallback (as new MVT())